tests/unit/test-util/test.adb.js

```diff
@@ -59,11 +67,13 @@ function getFakeADBKit(
 
   adbUtilReadAllStub.onCall(0).returns(Promise.resolve(Buffer.from('')));
 
+
   const fakeADBClient = {
     listDevices: sinon.spy(() => {
       return [];
     }),
     shell: sinon.spy(() => Promise.resolve('')),
+    readdir: sinon.spy(() => Promise.resolve([file])),
```

Comment:
I think that this spy could return an empty array by default (because for most of the test cases we don't care about this and an empty dir represents the scenario where no old artifacts dir have been found, which is a completely valid base scenario)